package.nls.json

```diff
@@ -38,5 +38,6 @@
     "azureApiManagement.copyDockerRunCommand": "Copy Docker Run Command",
     "azureApiManagement.generateKubernetesDeployment": "Generate Kubernetes Deployment File",
     "azureApiManagement.generateNewGatewayToken": "Generate New Gateway Token",
-    "azureApiManagement.debugPolicy": "Start Policy Debugging"
+    "azureApiManagement.debugPolicy": "Start Policy Debugging",
+    "azureApiManagement.generateFunctions": "Generate Functions from API"
```

Comment:
Change the name to Generate Azure Functions App

Reply:
"Generate Azure Functions" should do to keep it consistent with "Import Azure Functions"